examples/avro-cli.py

```diff
@@ -137,9 +156,12 @@ def consume(topic, conf):
                 print("Consumer error: {}".format(msg.error()))
                 continue
 
-            record = User(msg.value())
-            print("name: {}\n\tfavorite_number: {}\n\tfavorite_color: {}\n".format(
-                record.name, record.favorite_number, record.favorite_color))
+            record = msg.value()
```

Comment:
Add a comment that clarifies that record is indeed a User object and record_id is whatever it is.